docs/source/azure/scenarios/logicappnotification.rst

```diff
@@ -0,0 +1,166 @@
+.. _azure_scenarios_logic_app_notifications:
+
+Using Azure Logic Apps to notify users of policy violations.
+==========================================================
+
+Azure Logic Apps are a great option to create simple scalable workflows in Azure such as sending an email or posting to Microsoft Teams. Logic Apps can have a `Request trigger` to initiate a workflow and this can be invoked with a Custodian :ref:`logic-app <Azure.common.actions.logic-app>` action.
+
+Create Azure Logic App
+--------------------------------
+We will now walkthrough the steps to configure a Logic Apps workflow that will consist of an `Request trigger` and an `Outlook send email action`.
+
+Create a new Azure Logic App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. From the main Azure menu, choose **Create a resource ─▶ Integration ─▶ Logic App.**
+2. Under **Templates**, choose **Blank Logic App**.
```

Comment:
For me in the portal there were 2 extra steps between step 1 and step 2. It's probably OK for a typical Azure portal user, but in case it throws anyone off it might help to start step 2 with: 

'Inside the new Logic App resource, under **Templates** .... ' 

Or something similar that makes it clear it's inside the new resource and not part of creating the resource.